src/parsers/chase-credit-card-parser.ts

```diff
@@ -82,11 +82,7 @@ function performStateAction(currentState: State, line: string, yearPrefix: numbe
 
         const result = processTransactionLine(line, output.startDate, output.endDate);
 
-        if (typeof result === 'string') {
-            if (!!result) {
-                array[array.length - 1].description += '\n' + collapseSpaces(result);
-            }
-        } else {
+        if (typeof result !== 'string') {
```

Comment:
I don't see any transactions in the statements I have available to me that have multiple lines of description, so this is a good change.

Reply:
Oh wait, I was looking at the statements wrong. I actually _do_ have multi-line transaction descriptions. I'll add another state for page transitions (I don't have any statements available to me that span multiple pages though so I won't be able to test it myself).